rdkPlugins/GPU/source/GpuPlugin.cpp

```diff
@@ -62,13 +61,15 @@ unsigned GpuPlugin::hookHints() const
  *
  *  @return true if successful, otherwise false.
  */
-bool GpuPlugin::createRuntime()
+bool GpuPlugin::postInstallation()
 {
+    const std::string cgroupDirPath = getGpuCgroupMountPoint();
+
     // if we don't have a GPU cgroup controller then nothing to do
-    if (mCgroupDirPath.empty())
+    if (cgroupDirPath.empty())
     {
         AI_LOG_ERROR_EXIT("missing gpu cgroup directory");
-        return true;
+        return false;
```

Comment:
Why has this flipped?

Reply:
If the cgoup directory isn't present, we should return with an error.

We could return true as a sign of "oh, you don't have the cgroup so no worries!", but that's a bit misleading IMO.

An upcoming commit will change the one in `postHalt()` as well.